AppCenterData/AppCenterData/MSData.h

```diff
@@ -68,7 +68,9 @@ typedef void (^MSPaginatedDocumentsCompletionHandler)(MSPaginatedDocuments *docu
 + (void)readDocumentWithID:(NSString *)documentID
                       type:(Class)documentType
                  partition:(NSString *)partition
-         completionHandler:(MSDocumentWrapperCompletionHandler)completionHandler;
+         completionHandler:(MSDocumentWrapperCompletionHandler)completionHandler // clang-format off
+NS_SWIFT_NAME(read(withDocumentID:type:partition:completionHandler:));
```

Comment:
```suggestion
NS_SWIFT_NAME(read(withDocumentID:documentType:partition:completionHandler:));
```
I'm for consistency 🙂

Reply:
Sounds legit to me, actually it would make sense to add it in objective c too.